### ISSUE TITLE
[Feature Request] Add flag to cancel upload instead of using the default metadata

### ISSUE THREAD

**Issue by Xylot** [porjo/youtubeuploader#71]
It would be very useful to have the ability to disregard the upload instead of uploading with the video with the default metadata.

**Comment by porjo** [comment]
I'm not sure what you mean. Could you rephrase the question?

**Comment by Xylot** [comment]
In files.go lines 43-47

		if e != nil {
			fmt.Printf("Error reading file '%s': %s\n", filename, e)
			fmt.Println("Will use command line flags instead")
			goto errJump
		}

Is it possible to add a flag that just errors out of the upload instead of going to command-line flags? For reasons currently unknown to me, sometimes the uploader does not accept my metadata file and If that's the case I do not want the video to be uploaded.

**Comment by porjo** [comment]
Yes, I agree that failing to read the metadata file should be fatal. Rather than add a flag, I've made that the default behaviour. I've pushed [release 20.03](https://github.com/porjo/youtubeuploader/releases/tag/20.03) to fix that
